packages/desktop-client/src/components/accounts/MobileAccountDetails.js

```diff
@@ -95,7 +95,7 @@ export default function AccountDetails({
           alignItems: 'center',
           flexShrink: 0,
           overflowY: 'hidden',
-          paddingTop: 20,
+          paddingTop: 10,
```

Comment:
Aligning the `all accounts` and `single account` header location. If we don't do this, then the "plus" button jumps a few pixels when navigating between the two pages.

Reply:
Keeping this patch as it will be useful when we add account creation to mobile.